### ISSUE TITLE
gonum.org/v1/gonum doesn't take you anywhere useful

### ISSUE THREAD

**Issue by pwaller** [gonum/gonum#850]
At the moment if I visit the import path `gonum.org/v1/gonum` in the browser, it doesn't go anywhere useful.

It goes to a 404 pages which suggest a few things, but none of them point me to the git repository, for example.

![image](https://user-images.githubusercontent.com/438648/52634557-6b93f280-2ebf-11e9-8216-55834a773daa.png)

**Comment by kortschak** [comment]
Yes, I think we were aware of this, but had just not got around to it. It belongs in the website repo though, so I'll move it there.
